pkg/distributor/distributor_test.go

```diff
@@ -3481,7 +3658,7 @@ func TestDistributor_Push_Relabel(t *testing.T) {
 			})
 
 			// Push the series to the distributor
-			req := mockWriteRequest(tc.inputSeries, 1, 1)
+			req := mockWriteRequest(tc.inputSeries, 1, 1, false)
```

Comment:
Should we test for the histogram=true case as well?

Reply:
I didn't change it because relabel logic ideally has nothing to do with whether native histogram data or not.
But yeah I can do it.

Reply:
Updated: fixed in latest commit